projects/isy-angular-widgets-demo/src/app/feature/objekt-anzeigen/objekt-anzeigen.component.html

```diff
@@ -1,301 +1,352 @@
+          <div class="grid p-1 align-items-center">
+            <label for="credit-card-number" class="col-12 sm:col-4 md:col-5 lg:col-6 sm:text-right text-left">
+              {{ 'isyAngularWidgetsDemo.labels.kreditkartennummer' | translate }}
             </label>
+            <div class="col-12 sm:col-8 md:col-7 lg:col-6">
+              <input id="credit-card-number" formControlName="creditCardNumber" class="w-full" type="text" pInputText />
+            </div>
           </div>
-          <div class="col-5">
-            <p-inputNumber id="security-level" formControlName="securityLevel" [showButtons]="true" [min]="0" [max]="9">
-            </p-inputNumber>
+        </div>
+        <div class="col-12 lg:col-6 xl:col-4 py-0">
+          <div class="grid p-1 align-items-center">
+            <label for="credit-card-expiration-date" class="col-12 sm:col-4 md:col-5 lg:col-6 sm:text-right text-left">
+              {{ 'isyAngularWidgetsDemo.labels.ablaufdatumKreditkarte' | translate }}
+            </label>
+            <div class="col-12 sm:col-8 md:col-7 lg:col-6">
+              <input
+                id="credit-card-expiration-date"
+                formControlName="creditCardExpirationDate"
+                class="w-full"
+                type="text"
+                pInputText
+              />
+            </div>
           </div>
-          <div class="col-6"></div>
         </div>
-        <div class="grid m-0">
-          <div class="col-1 text-right align-self-center">
-            <label for="intelligence-notes">
-              {{ 'isyAngularWidgetsDemo.labels.geheimdienstnotizen' | translate }}
+        <div class="col-12 lg:col-6 xl:col-4 py-0">
+          <div class="grid p-1 align-items-center">
+            <label for="identity-document" class="col-12 sm:col-4 md:col-5 lg:col-6 sm:text-right text-left">
+              {{ 'isyAngularWidgetsDemo.labels.identityDocument' | translate }}
             </label>
+            <div class="col-12 sm:col-8 md:col-7 lg:col-6">
+              <p-fileUpload
+                id="identity-document"
+                formControlName="identityDocument"
+                ngDefaultControl
+                [name]="'myfile'"
+                [accept]="'image/*,.pdf,.txt'"
+                [auto]="true"
+                [maxFileSize]="1000000"
+                [customUpload]="true"
+                (uploadHandler)="uploadFile($event)"
+              >
+              </p-fileUpload>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="field field-checkbox my-5">
+        <p-inputSwitch inputId="show-secret-fields" (onChange)="showSecretFields = $event.checked"></p-inputSwitch>
+        <label for="show-secret-fields" class="m-0 ml-3">
+          {{ 'isyAngularWidgetsDemo.actions.showSecretFields' | translate }}
+        </label>
+      </div>
+
+      <div id="div-show-secret-fields" *ngIf="showSecretFields">
+        <div class="grid m-0 align-items-center">
```

Comment:
align-items-center is necessary because otherwise the input fields are no longer aligned horizontally correctly when using long labels that extend over several lines and the checkbox is also not displayed vertically in the center.
Changing the margin to m-1 inserts a gap not only at the top and bottom but also on the left and right. As a result, the fields are no longer flush with the upper fields.